.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*    @dialoguemd/devops
```

Comment:
Not sure that we are owners of this lib

Reply:
🤷 Ed sent me a list of the ones he thought we could add to our team. Any thoughts on this @edouardlp ?

Reply:
did not know that it was coming from Ed, alright then